Dockerfile

```diff
@@ -13,8 +13,9 @@ COPY poetry.lock pyproject.toml /
 # PRODUCTION
 #####
 FROM base as production
-ENTRYPOINT poetry run gunicorn -b 0.0.0.0:5000 --chdir /src wsgi
-RUN poetry install --no-dev --no-root
+ENTRYPOINT ["poetry run gunicorn -b 0.0.0.0:$PORT --chdir /src wsgi"]
```

Comment:
Two thoughts here:
- This change from a hardcoded port value means that your instructions for running the prod container locally aren't valid any longer & the container fails to start, we should fix that (probably either by defining a default port value here in the dockerfile, or by passing one in from `docker-compose.yml`)
- The exec form of entrypoint expects an array of strings passing the command and all arguments separately, and indeed is causing another layer of issues when running this. It will also fail to substitute the `$PORT` variable - one solution here would be to start a bash script from entrypoint that then runs the start command for the container